include/roboteam_ai/stp/Play.hpp

```diff
@@ -61,7 +61,7 @@ class Play {
     /**
      * Default ctor, ensures proper construction of Play
```

Comment:
It's not default anymore
```suggestion
     * Ctor that constructs a play an assigns its name
```